packages/ethereum/contracts/Makefile

```diff
@@ -5,7 +5,8 @@ FOUNDRY_DIR ?= ${CURDIR}/../../../.foundry
 PATH := $(subst :${FOUNDRY_DIR}/bin,,$(PATH)):${FOUNDRY_DIR}/bin
 
 # use custom PATH in all shell processes
-SHELL := env PATH=$(PATH) $(shell which bash)
+#SHELL := env PATH=$(subst $(space),\$(space),$(PATH)) $(shell which bash)
+SHELL := /bin/bash
```

Comment:
Why this change?

Reply:
Ah, good catch. Used it as a temporary to make stuff work on my Mac. Will revert it